qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -942,10 +935,10 @@ def new_x_concatenated_sequence(
 
     values, counts = np.unique(pos_cum_sum, return_counts=True)
 
-    offsets = [values[i] for i in range(counts.shape[0]) if counts[i] % 2 == 0]
+    offsets = [values[idx] for idx, count in enumerate(counts) if count % 2 == 0]
```

Comment:
optional: you could also do `[value for value, count in zip(values, counts) if count % 2 == 0]`

Reply:
nice! Can you approve again. Just apply the changes.